### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,33 +11,33 @@
...
         </header> 
       </div>
-      
-        
-       
      
         <article>
-          <div><img src="images/header.jpg" class="header-pic" alt=""></div>
-        
           <section id="about_me"><div class="about-me-wrap">
```
Comment (ttaannjjaa) [inline index.html:36]: no code problem , i think this happened randomly not beginning a new row with the `div`tag

File: index.html [hunk 2]
```diff
@@ -56,29 +56,77 @@ <h2>Contact:</h2>
...
           </div>
 
           <h2>Connect with me</h2>
           <div id="connect"class="connect">
-            <section><a href="https://github.com/FaGru" ><img src="images/GitHub-white.png"class ="button-connect" alt=""></a></section>
-            <section><a href="CodePen XXXXXXXXXXXXXXXX"><img src="images/codepen.png"class ="button-connect" alt=""></a></section>
-            <section><a href="LinkedIn XXXXXXXXXXXXXXX" ><img src="images/LinkedIn.png"class ="button-connect" alt=""></a></section>
+            <section><a href="https://github.com/FaGru" ><img src="images/GitHub-white.png"class ="button-connect" alt="Link Github"></a></section>
+            <section><a href="CodePen XXXXXXXXXXXXXXXX"><img src="images/codepen.png"class ="button-connect" alt="Link CodePen"></a></section>
+            <section><a href="LinkedIn XXXXXXXXXXXXXXX" ><img src="images/LinkedIn.png"class ="button-connect" alt="Link"></a></section>
```
Comment (ttaannjjaa) [inline index.html:78]: just a feeling 😃...originally you wanted to add   LinkedIn   in the `alt`-attribute

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Really cool design - great work 👍😃

File: index.html (hunk 1 above)
Comment (FaGru) [reply index.html:36]: that's right thx :)

File: index.html (hunk 2 above)
Comment (FaGru) [reply index.html:78]: oh yes i think i worked to fast there 😅

**Pull request closed without merging**